compiler/main/driver.cpp

```diff
@@ -1780,6 +1781,16 @@ static void validateSettings() {
   checkRuntimeBuilt();
 }
 
+static void setupDynoCompilerFlags() {
+  chpl::CompilerFlags compFlags = chpl::CompilerFlags();
+  compFlags.set(chpl::CompilerFlags::DEVELOPER, developer);
+  // TODO: Setting this flag causes a bunch of warnings to be printed
+  // but they are coming from our internal modules, so we should squelch them
+  // Seems to be coming from frontend/lib/uast/post-parse-checks.cpp:818
+  // compFlags.set(chpl::CompilerFlags::WARN_UNSTABLE, fWarnUnstable);
```

Comment:
Yeah, we'll have to apply what I believe Lydia has been working on to silence warnings from internal/standard module code.